README.md

```diff
@@ -128,17 +128,21 @@ end
 `Fail2Ban.filter` can be used within a blacklist to block all requests from misbehaving clients.
 This pattern is inspired by [fail2ban](http://www.fail2ban.org/wiki/index.php/Main_Page).
 See the [fail2ban documentation](http://www.fail2ban.org/wiki/index.php/MANUAL_0_8#Jail_Options) for more details on
-how the parameters work.
+how the parameters work.  All fail2ban filters share the same match counter, using one filter per app is recommended.
```

Comment:
Yes!  In testing I tried adding a `Rack::Attack.blacklist('fail2ban wp')` block and a `Rack::Attack.blacklist('fail2ban passwd')` block, each one using a fail2ban filter but checking different criteria.  The cache keys written for both looked like `rack::attack:2409775:fail2ban:count:127.0.0.1` and matching a filter in the first block would also increment the counter for the second block.  Eventually when one of them was triggered it would add a ban rule to the cache for the IP address and it was logged as the first matching blacklist.  What I got from that (and a skimming the fail2ban filter code) was that the fail2ban filters do not include the rack-attack blacklist name in any cache keys and instead they all use the same cache keys for counts and bans.  Changing that would make a fine PR, but trying to explain how it works now in the docs was quick.